cardano-api/src/Cardano/Api/Address.hs

```diff
@@ -606,4 +615,3 @@ fromShelleyStakeReference (Shelley.StakeRefPtr ptr) =
   StakeAddressByPointer (StakeAddressPointer ptr)
 fromShelleyStakeReference Shelley.StakeRefNull =
   NoStakeAddress
```

Comment:
I think your editor is automatically removing newlines at the end of the document.

Reply:
Yeah it's adding unnecessary diff to the PR, so it would be good if you can turn it off to avoid this.